templates/bootstrap.sh

```diff
@@ -463,7 +463,7 @@ else
     fi
 
     # Run the build script command in the service specified in BUILDKITE_DOCKER_COMPOSE_CONTAINER
-    echo "+++ $BUILDKITE_COMMAND_ACTION (in Docker Compose container)"
+    echo "~~~ $BUILDKITE_COMMAND_ACTION (in Docker Compose container)"
```

Comment:
I recently added this in https://buildkite.com/changelog/10-buildkite-agent-2-dot-1-9 — perhaps we need to update the other parts of the bootstrap to do this instead?